subprojects/packagefiles/google-highway/meson.build

```diff
@@ -1,7 +1,7 @@
 project(
   'highway',
   'cpp',
-  version: '1.2.0',
+  version: 'fa78bde6a8979031b4ed5a68da7d8cfa1d4255b8c41f49c335b4d7ea402fb769',
```

Comment:
We generally do not package random git snapshots, unless the upstream project had decided not to make releases anymore. The documentation on the main git page suggests that highway is using semver, so I'm not sure why we'd move to a git snapshot.